src/components/Player.js

```diff
@@ -35,6 +36,7 @@ const propTypes = {
   loop: PropTypes.bool,
   autoPlay: PropTypes.bool,
   src: PropTypes.string,
+  type: PropTypes.oneOf(['audio', 'video']),
```

Comment:
it might be nice to export these from the player itself so we dont have to worry about typos. similar to how we export flavors from the Button component